source/includes/custom-field-options/_update.md.erb

```diff
@@ -0,0 +1,58 @@
+## Update Custom Field Option
+
+```http
+PATCH https://api.teamtailor.com/v1/custom-field-values/1 HTTP/1.1
+Authorization: Token token=abc123abc123
+X-Api-Version: <%= config[:api_version] %>
+
+{
+  "data": {
+    "id": 1,
+    "type": "custom-field-options",
+    "attributes": {
+      "value": "Updated value",
+    }
+  }
+}
+```
+
+```shell
+curl -X "PATCH" "https://api.teamtailor.com/v1/custom-field-values/1" \
+     -H "Authorization: Token token=abc123abc123" \
+     -H "X-Api-Version: <%= config[:api_version] %>"
+     -H "Content-Type: application/vnd.api+json" \
+     -d '{
+   "data": {
+     "id": 1,
+     "type": "custom-field-options",
```

Comment:
Is this the correct type, shouldn't it be `custom-field-values`?

Reply:
no, this is correct. It updates the custom field option value. not the selected value from custom field.